holoviz/pyviz_comms
# Update to JupyterLab 4

This PR upgrades pyviz_comms to be a jupyterlab4 extension. I've tested it by ensuring it builds successfully, the tests pass, and that I can run holoviz with it. Are there other tests I should run?

A couple of the things I am unsure about:
- How to test the conda recipe
- If the new workflow files generated should be included (I have done so, but I can remove them if need be)
- Whether the following files should be removed:
    - dodo.py
    - install.json

Thanks!
